plugins/nodes/src/js/components/NodeBreadcrumbs.js

```diff
@@ -19,22 +18,15 @@ const NodeBreadcrumbs = ({ nodeID, taskID, taskName, unitID }) => {
       </BreadcrumbTextContent>
     </Breadcrumb>
   ];
-  let node;
 
-  if (nodeID != null && trimmedNodeID.length > 0) {
-    node = CompositeState.getNodesList()
-      .filter({ ids: [nodeID] })
-      .last();
-
-    if (node) {
-      crumbs.push(
-        <Breadcrumb key="hostname" title={node.hostname}>
-          <BreadcrumbTextContent>
-            <Link to={`/nodes/${encodedNodeID}`}>{node.hostname}</Link>
-          </BreadcrumbTextContent>
-        </Breadcrumb>
-      );
-    }
+  if (node) {
+    crumbs.push(
+      <Breadcrumb key="hostname" title={node.hostname}>
+        <BreadcrumbTextContent>
+          <Link to={`/nodes/${encodedNodeID}`}>{node.hostname}</Link>
+        </BreadcrumbTextContent>
+      </Breadcrumb>
+    );
   }
 
   if (taskID != null && taskName != null) {
```

Comment:
should we add `node && ` to this if? if the `node === null` then `encodedNodeID = "null"` based on the code on lines 12 & 13. 

Or we could move this if block inside the `if (node)` block on line 22. Then we could also move the assignment of `trimmedNodeID` & `encodedNodeID` to the top of the same block.